deps/utf8proc.version

```diff
@@ -1,2 +1,2 @@
-UTF8PROC_BRANCH=v1.3
-UTF8PROC_SHA1=85789180158ac7fff85b9f008828d6ac44f072ea
+UTF8PROC_BRANCH=v2.0.1
+UTF8PROC_SHA1=a1fe9955bbc75ffb923c1219bf58befd2688e34c
```

Comment:
it should happen automatically in configure-utf8proc or one of those, as long as you aren't downloading it in git mode (which is not the default any more)